examples/README.md

```diff
@@ -0,0 +1,14 @@
+# Azure product naming and what example to use:
+
+Today (February 2025) the production, general availability Confidential Containers solution in Azure is Confidential ACI (Azure Container Instances). It is available with two sorts of orchestration, direct ACI (create container groups one at a time and manage them youself) and with AKS using "virtual nodes on Azure Container Instances".
+
+There is a preview of AKS Confidential Pods which is based on the Kata scheme.
+
+Some details of how the ACI and Kata based solution are differnet. These include items such as how yto invoke the az confcom policy tool and how various data is passed for the runtime into the container.
```

Comment:
*some aspects of the ACI and Kata-based solution are different (different has a typo as well)

*how to invoke (typo)

Reply:
Done

Reply:
*how to invoke (typo) was not fixed